internal/server/http/handler_test.go

```diff
@@ -69,6 +71,59 @@ func TestImposterHandler(t *testing.T) {
 	}
 }
 
+func TestImposterHandler_Variables(t *testing.T) {
+	var headers = make(map[string]string)
+	headers["Content-Type"] = "application/json"
+
+	responseId1 := "test/testdata/imposters_variables/responses/gopher_1_response.json"
+	responseId2 := "test/testdata/imposters_variables/responses/gopher_2_response.json"
+	responseId1Variable1 := "test/testdata/imposters_variables/responses/gopher_1_1_response.json"
+	responseId1Variable2 := "test/testdata/imposters_variables/responses/gopher_1_2_response.json"
+
+	imposterFilePath := "test/testdata/imposters_variables/gopher_variables.imp.json"
+	imposterFile, _ := os.Open(imposterFilePath)
+	defer imposterFile.Close()
+	imposterBytes, _ := io.ReadAll(imposterFile)
+
+	var imposters []Imposter
+	err := json.Unmarshal(imposterBytes, &imposters)
+	assert.NoError(t, err)
+
+	var dataTest = []struct {
+		name             string
+		imposter         Imposter
+		url              string
+		expectedBodyPath string
+		statusCode       int
+	}{
+		{"valid imposter with id 1 in path", imposters[0], "/gophers/1", responseId1, http.StatusOK},
```

Comment:
Can you add a test case where there's a value defined in the `bodyFile` path but not in the `endpoint`, please? Just to reflect what's the expectation in such case.

Reply:
for sure.. 
but this is the same behaviour as right now if the file could not be found 

"200" with empty body